collect_app/src/main/java/org/odk/collect/android/audio/AudioControllerView.java

```diff
@@ -65,7 +65,11 @@ public AudioControllerView(Context context, AttributeSet attrs) {
         seekBar.setOnSeekBarChangeListener(swipeListener);
 
         binding.play.setOnClickListener(view -> playClicked());
-        binding.remove.setOnClickListener(view -> listener.onRemoveClicked());
+        binding.remove.setOnClickListener(view -> {
```

Comment:
Were you able to reproduce this crash? If it's possible to click the button before setting the listener, then it should be possible with the play button as well, right?